src/app/shared/components/template/components/layout/popup/popup.component.scss

```diff
@@ -18,7 +18,7 @@
     .popup-container {
       height: var(--safe-area-height);
     }
-    .close-button { 
+    .close-button {
```

Comment:
Not sure why this is cropping up now, but reverting it and committing isn't possible as the linter removes the whitespace again when I attempt a commit